rest_framework/schemas/openapi.py

```diff
@@ -106,7 +118,9 @@ def get_operation(self, path, method):
             operation['requestBody'] = request_body
         operation['responses'] = self._get_responses(path, method)
 
-        return operation
+        component_schema = self._get_component_schema(path, method)
+
+        return operation, component_schema
```

Comment:
I see why you've done this, but I think it would be better to refactor to avoid the tuple return here. c.f. #7127.

Reply:
Thanks ! I rebased my PR on your's !